tests/libdv/dv_builder.go

```diff
@@ -0,0 +1,132 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2020 Red Hat, Inc.
+ *
+ */
+
+package libdv
+
+import (
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/rand"
+	"kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1"
+)
+
+const (
+	dvRandomNameLength       = 12
+	PVCSizeForRegistryImport = "512Mi"
+)
+
+// dvOption is an option type for the NewDataVolume function
+type dvOption func(*v1beta1.DataVolume)
+
+// NewDataVolume Set up a new DataVolume with a random name, a namespace and an optional list of options
+func NewDataVolume(options ...dvOption) *v1beta1.DataVolume {
+	name := "test-datavolume-" + rand.String(dvRandomNameLength)
```

Comment:
Make it similar to this?
https://github.com/kubevirt/kubevirt/blob/b94cd3221355b2dcf400b61161f6daaac46bd050/tests/libvmi/vmi.go#L49